org.eclipse.wb.swt/src/org/eclipse/wb/internal/swt/model/property/editor/color/ColorPropertyEditor.java

```diff
@@ -315,6 +313,7 @@ private static String getSource(GenericProperty property, ColorInfo colorInfo) t
     }
     IPreferenceStore preferences =
         property.getJavaInfo().getDescription().getToolkit().getPreferences();
+    preferences.addPropertyChangeListener(null);

```

Comment:
> preferences.addPropertyChangeListener(null);

What is the intended use of this? Are you using it for its side-effects? 

![image](https://user-images.githubusercontent.com/180969/142436650-d7681e8e-d5bf-41a0-89f7-62d551fe201b.png)